dtreeviz/colors.py

```diff
@@ -24,7 +27,61 @@
     ['#FEFEBB', '#edf8b1', '#c7e9b4', '#7fcdbb', '#1d91c0', '#225ea8', '#fdae61', '#f46d43'],  # 8
     ['#FEFEBB', '#c7e9b4', '#41b6c4', '#74add1', BLUE, DARKBLUE, LIGHTORANGE, '#fdae61', '#f46d43'],  # 9
     ['#FEFEBB', '#c7e9b4', '#41b6c4', '#74add1', BLUE, DARKBLUE, LIGHTORANGE, '#fdae61', '#f46d43', '#d73027']  # 10
+
 ]
+def get_colorblind_friendly_colors():
```

Comment:
@tlapusan did this get merged in? I don't see this in colors.py

Reply:
@parrt it should be. Looking into the master code, we support the case when the number of classes is big. Implemented by get_hex_colors() method.